webpack.config.js

```diff
@@ -149,8 +149,8 @@ const config = (env, argv) => ({
       filename: 'index.html',
     }),
     new webpack.DefinePlugin({
-      // use appropriate (dev or production) PP settings
-      PP_SETTINGS: appSettings[argv.mode]
+      // use appropriate (development or production) PP settings
+      PP_SETTINGS: JSON.stringify(appSettings[argv.mode]),
```

Comment:
Oh, I haven't noticed that and was thinking about something a bit different, but it doesn't matter. It does the job well!